test/unittests/mongoBackend/mongoRegisterContext_update_test.cpp

```diff
@@ -183,7 +186,8 @@ TEST(mongoRegisterContext_update, updateCase1)
   prepareDatabase();
 
   /* Invoke the function in mongoBackend library */
-  ms = mongoRegisterContext(&req, &res);
+  ms = mongoRegisterContext(&req, &res, "", "/");
+  LM_M(("Invoke mongoRegisterContext in mongoBackend library and EXIT. ms: %d", ms));
```

Comment:
Garbage.
We have lots of LM_M in the unittest code so when I search for LM_Ms in the code I do a 'grep -v unittest', that's why I didn't find this line.
I will remove it